package.json

```diff
@@ -66,14 +66,13 @@
   },
   "dependencies": {
     "@grpc/grpc-js": "^1.3.3",
-    "@protobuf-ts/grpcweb-transport": "2.5.0",
-    "@protobuf-ts/runtime": "2.5.0",
-    "@protobuf-ts/runtime-rpc": "2.5.0",
-    "ldk": "0.4.12",
-    "slip77": "0.1.1",
+    "@protobuf-ts/grpcweb-transport": "^2.6.0",
+    "@protobuf-ts/runtime": "^2.6.0",
+    "@protobuf-ts/runtime-rpc": "^2.6.0",
+    "ecpair": "^2.0.1",
+    "ldk": "0.5.0",
+    "slip77": "0.2.0",
+    "tiny-secp256k1": "2.2.1",
```

Comment:
I'm thinking about this. Maybe we shouldn't depend on tiny-secp256k1 but use the "injectable pattern" (mainly if we don't want to use pure wasm on mobile).

Slip77 is a good example of an "injectable ecc pattern" https://github.com/vulpemventures/slip77/blob/master/ts_src/index.ts

Reply:
@tiero What do you think?